pkg/log/hclog.go

```diff
@@ -0,0 +1,58 @@
+package log
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+
+	"github.com/hashicorp/go-hclog"
+	"google.golang.org/grpc/metadata"
+	"gopkg.in/natefinch/lumberjack.v2"
+
+	"kusionstack.io/kusion-module-framework/pkg/util/kfile"
+)
+
+const (
+	KusionModuleName = "KUSION_MODULE_NAME"
+	KusionTraceID    = "KUSION_TRACE_ID"
+)
+
+func GetModuleLogger(ctx context.Context) hclog.Logger {
+	traceID := getTraceID(ctx)
+	moduleName := getModuleName(ctx)
+	if moduleName == "" {
+		moduleName = "default"
+	}
+
+	kusionDataDir, _ := kfile.KusionDataFolder()
+	logFile := filepath.Join(kusionDataDir, Folder, "modules", moduleName, fmt.Sprintf("%s.log", moduleName))
```

Comment:
Perhaps we need to add a check to verify whether the log file exists here? And if it doesn't exist, we should create one.

Reply:
lumberjackLogger will  create the absolute log path.